cogs/birthday.py

```diff
@@ -0,0 +1,126 @@
+import discord
+from discord import app_commands
+from discord.ext import tasks, commands
+import config
+import asyncio
+import datetime
+from datetime import date
+import os
+import pytz
+import requests
+
+AIRTABLE_API_KEY = os.environ['UPE_AIRTABLE_API_KEY']
+AIRTABLE_BASE_ID = os.environ['UPE_AIRTABLE_BASE_KEY']        
+AIRTABLE_TABLE_ID = os.environ['UPE_AIRTABLE_TABLE_ID']
+BIRTHDAY_FIELD_ID = 'fldoUSAvFtdsI51FP'
+DISCORD_NAME_FIELD_ID = 'fldoUSAvFtdsI51FP'
+NAME_FIELD_ID = 'fld3mtI8hayU9UPHD'
+YELLOW_COLOR = 0xFFBF00  
+RED_COLOR = 0xD2222D
+
+BOT_KEY = os.environ['BOT_KEY']
+
+class Birthday(commands.Cog):
+    '''
+    Checks 8am every day for birthdays
+    If there are birthdays @marketing
+    '''
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.MEMBERSHIP_CHANNEL_ID = 1061194622472298559 if config.isProd else 1065042160426176632 
```

Comment:
I know Ivan has this conditional logic for differentiating between the Prod and Dev environments, but could we choose to instead load these IDs from a config? That way we can just load a Dev environment config for local testing instead of having to update these all across the codebase.

Ideally there should be a followup task to clean up that tech debt but it's probably best we start this pattern now,

Reply:
Yeah, there are constants for pretty much everything... scattered everywhere repeatedly.  That should be extracted. I totally getcha but I think for this one cog it is fine so Gino can land this. I'll do the cleaning.